aikau/src/main/resources/alfresco/forms/controls/SitePicker.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Copyright (C) 2005-2015 Alfresco Software Limited.
+ *
+ * This file is part of Alfresco
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/**
+ * <p>A form control for allowing the user to select sites from the Alfresco repository.</p>
+ *
+ * @example <caption>Sample usage:</caption>
+ * {
+ *    name: "alfresco/forms/Form",
+ *    config: {
+ *       okButtonPublishTopic: "SITES_PICKED",
+ *       okButtonLabel: "OK",
+ *       widgets: [
+ *          {
+ *             name: "alfresco/forms/controls/SitePicker",
+ *             config: {
+ *                id: "SITE_PICKER",
+ *                name: "site",
+ *                description: "Pick a site, any site",
+ *                label: "Site",
+ *                multiSite: false
+ *             }
+ *          }
+ *       ]
+ *    }
+ * }
+ *
+ * @module alfresco/forms/controls/SitePicker
+ * @extends module:alfresco/forms/controls/Picker
+ * @author Dave Draper
+ */
+define(["alfresco/forms/controls/Picker",
+        "dojo/_base/declare"],
+        function(Picker, declare) {
+
+   return declare([Picker], {
+
+      /**
+       * An array of the i18n files to use with this widget.
+       *
+       * @instance
+       * @type {Object[]}
+       */
+      i18nRequirements: [
+         {i18nFile: "./i18n/Picker.properties"},
```

Comment:
No, it doesn't work like that... the i18nRequirements and the cssRequirements are only used when Surf analyses the source of a widget to work out its dependencies (this is why you can't configure in additional CSS and i18n dependencies are in a WebScript controller). So because the parent Picker is recognized as a dependency, its i18n dependencies will be included automatically.